package-lock.json

```diff
@@ -26,15 +24,18 @@
         "react-firebase-hooks": "^5.1.1",
         "react-icons": "^4.8.0",
         "react-redux": "^8.0.5",
+        "react-toastify": "^9.1.3",
```

Comment:
Don't push `package-lock.json` file if you haven't installed any new package in `package.json`